src/kibana/require.config.js

```diff
@@ -25,6 +25,7 @@ require.config({
     faker: 'bower_components/Faker/faker',
     file_saver: 'bower_components/FileSaver/FileSaver',
     gridster: 'bower_components/gridster/dist/jquery.gridster',
+    heat: 'bower_components/Leaflet.heat/dist/leaflet-heat',
```

Comment:
It would make sense to namespace this to leaflet-heat, like we do with all angular plugins.